src/py_server/requirements.txt

```diff
@@ -0,0 +1 @@
+Flask==2.0.3
```

Comment:
Please add install instructions to the readme

Reply:
Done